src/race.js

```diff
@@ -0,0 +1,51 @@
+function showResult(cars, repeatNum) {
+  const location = repeatNum == 0 ? '#result' : '#result_' + (repeatNum - 1);
+  const resultHead = document.getElementById(location);
+  resultHead.insertAdjacentHTML(
+    "afterend",
+    `<div id=#result_${repeatNum}></div>`
+  );
+  let ret = "";
+  cars.forEach(item => {
+    let scores = "";
+    for (let i = 0; i < item._score; i++) {
+      scores += "-";
+    }
+    ret += `${item._name}  :  ${scores} <br>`;
+  });
+  ret += '<br>'
+  const newDiv = document.getElementById("#result_" + repeatNum);
+  newDiv.insertAdjacentHTML('beforeend', ret);
+}
+
+function showChampion(cars, inputNum) {
+  let finalScore = cars.map(x => x._score);
+  let highestScore = Math.max.apply(null, finalScore);
```

Comment:
```suggestion
  let highestScore = Math.max(...finalScore);
```
스프레드 연산자로 조금 더 간편하게 할 수도 있을것같습니다.

Reply:
멋져요..